custom/icds_reports/tasks.py

```diff
@@ -650,3 +657,46 @@ def collect_inactive_awws():
     sync.store_file_in_blobdb(export_file)
     sync.save()
     celery_task_logger.info("Ended updating the Inactive AWW")
+
+
+@periodic_task(run_every=crontab(minute=0, hour=0), queue='background_queue')
+def push_missing_docs_to_es():
+    current_date = date(2017, 1, 1)
+    # this interval may need to be reduced, particularly for the recent months
+    interval = timedelta(weeks=1)
+    case_doc_type = 'CommCareCase'
+    xform_doc_type = 'XFormInstance'
+    while current_date <= date.today() + interval:
```

Comment:
should this stop running if it's still going by 6am (or whatever time)?

Reply:
I think this is fine now because it's only comparing document counts. I am delaying the comparison of doc ids and resaving to another thask which is much more expensive